terraform/README.md

```diff
@@ -0,0 +1,76 @@
+<!-- BEGIN_TF_DOCS -->
+
+This repository contains the machine charm Terraform bundle for the Charmed Apache Kafka, available on [Charmhub](https://charmhub.io/kafka).
+
+The Charmed Apache Kafka Terraform bundle can be used to deploy Apache Kafka brokers and KRaft controllers, Kafka Connect, Karapace Schema Registry, and Kafbat Kafka UI with minimal configuration overhead.
+
+For a quick start, please refer to the `examples` folder which contains sample deployment scenarios.
+
+## Requirements
+
+| Name | Version |
+|------|---------|
+| <a name="requirement_terraform"></a> [terraform](#requirement\_terraform) | >=1.7.3 |
+| <a name="requirement_juju"></a> [juju](#requirement\_juju) | >=0.20.0 |
+
+## Providers
+
+| Name | Version |
+|------|---------|
+| <a name="provider_juju"></a> [juju](#provider\_juju) | 0.23.0 |
+
+## Modules
+
+| Name | Source | Version |
```

Comment:
Question: The contents of the last column don't look like a version number. Do you think we can come up with a better name? Like track or branch, maybe.